ApiBundle/Tests/Transformer/ContentTransformerTest.php

```diff
@@ -85,6 +84,10 @@ public function testTransform()
 
     /**
      * test reverseTransform
+     * @param FacadeInterface  $facade
```

Comment:
you should either : 
 - let one empty line between the comment and the first @param
 - suppress the comment